### DISCUSSION TITLE
When will API 34 be released?

### DISCUSSION THREAD

**Discussion by philipswe** [matomo-org/matomo-sdk-android#514] [General]
Hi,
I noticed that Android Target SDK 34 has been merged to master from @hannesa2 but when will it be included in a release? We need this asap in our project to build a compliant Android app, very soon Google won't accept apps built with less than SDK 34. 
Thanks in advance!

**Comment by hannesa2** [comment]
Here it is https://github.com/matomo-org/matomo-sdk-android/releases/tag/4.3
